ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,32 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+
+namespace PalindromeChecker {
+   class Program {
+      static bool IsPalindrome (string input) {
+         input = input.ToLower ().Replace (" ", "").Replace (".", "").Replace (",", "").Replace ("?", "").Replace ("'", "").Replace ("!", "");

```

Comment:
Do we need to remove these special characters from the input string?

Reply:
In the text mentioned in example of assignment the sentence was don't nod so only I thought the need to remove the punctuations from sentence

Reply:
Yes, the assignment states that the input can be a full sentence with special characters. You just have to check if the entire input string is a palindrome.